test/test_svd.cc

```diff
@@ -199,6 +220,18 @@ void test_svd_work( Params& params, bool run )
         // compute and save timing/performance
         params.time() = time;
 
+        if (timer_level >= 2) {
+            params.time2() = slate::timers[ "svd::geqrf" ];
+            params.time3() = slate::timers[ "svd::gelqf" ];
+            params.time4() = slate::timers[ "svd::ge2tb" ];
+            params.time5() = slate::timers[ "svd::tb2db" ];
+            params.time6() = slate::timers[ "svd::bdsqr" ];
+            params.time7() = slate::timers[ "svd::unmtr_hb2st" ];
+            params.time8() = slate::timers[ "svd::unmbr_ge2tb" ];
+            params.time9() = slate::timers[ "svd::unmqr" ];
+            params.time10() = slate::timers[ "svd::unmlq" ];
```

Comment:
You may need to put `if (wantu) {` around `unmbr_*_U` and `unmqr` timers `}`, and `if (wantv) {` around `unmbr_*_V` and `unmlq` timers `}`.

Reply:
Not needed, since a non-existent key returns 0.